.github/workflows/python-package.yml

```diff
@@ -198,6 +204,31 @@ jobs:
         name: bdist_wheel_${{ matrix.os }}_${{ matrix.arch }}_${{ matrix.python-version }}
         path: dist/bitsandbytes-*.whl
         retention-days: 7
+
+  audit-wheels:
+    needs: build-wheels
+    runs-on: ubuntu-latest
+    env:
+      PIP_DISABLE_PIP_VERSION_CHECK: 1
+    steps:
+    - uses: actions/checkout@v4
+    - name: Download all wheel artifacts
+      uses: actions/download-artifact@v2
```

Comment:
You can't use download-artifact@v2 with upload-artifact@v4, which is probably why you're not getting anything here :)